File: Beecrowd/1080 Maior e Posição/main.c
#include <stdio.h>
 
int main() {
 
	int X, maior = 0, i, cont = 0;
	
	for(i = 1; i <= 100; i++){
		scanf("%d", &X);
		if (i == 0) {
      		maior = X;
    	}
		else {
			if (X > maior) {
				maior = X;
				cont = i;
			}
		}
	}
	printf("%d\n", maior);
	printf("%d\n", cont);
 
    return 0;
}

File: Beecrowd/1080 Maior e Posição/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);

        int x, maior = 0, cont = 0;

        for (int i = 1; i <= 100; i++) {
            x = sc.nextInt();
            if (i == 0) {
                maior = x;
            } else {
                if (x > maior) {
                    maior = x;
                    cont = i;
                }
            }
        }
        System.out.println(maior);
        System.out.println(cont);

        sc.close();
    }
}